protocols/v2/subprotocols/mining/src/set_custom_mining_job.rs

```diff
@@ -6,87 +6,91 @@ use binary_sv2::{Deserialize, Seq0255, Serialize, Str0255, B0255, B064K, U256};
-    /// consensus. The general purpose bits (as specified in
-    /// BIP320) can be freely manipulated by the downstream
-    /// node.
+    /// Version field that reflects the current network consensus.
+    ///
+    /// The general purpose bits (as specified in BIP320) can be freely manipulated by the
+    /// downstream role. The downstream role must not rely on the upstream role to set the BIP320
+    /// bits to any particular value.
     pub version: u32,
-    /// Previous block’s hash, found in the block header field.
+    /// Previous block’s hash.
     #[cfg_attr(feature = "with_serde", serde(borrow))]
     pub prev_hash: U256<'decoder>,
-    /// Smallest nTime value available for hashing.
+    /// Smallest `nTime` value available for hashing.
     pub min_ntime: u32,
     /// Block header field.
     pub nbits: u32,
-    /// The coinbase transaction nVersion field.
+    /// The coinbase transaction `nVersion` field.
     pub coinbase_tx_version: u32,
-    /// Up to 8 bytes (not including the length byte) which are
-    /// to be placed at the beginning of the coinbase field in
-    /// the coinbase transaction.
+    /// Up to 8 bytes (not including the length byte) which are to be placed at the beginning of
+    /// the coinbase field in the coinbase transaction.
     pub coinbase_prefix: B0255<'decoder>,
     /// The coinbase transaction input’s nSequence field.
     pub coinbase_tx_input_n_sequence: u32,
-    /// The value, in satoshis, available for spending in
-    /// coinbase outputs added by the client. Includes both
-    /// transaction fees and block subsidy.
+    /// The value, in satoshis, available for spending in coinbase outputs added by the client.
+    /// Includes both transaction fees and block subsidy.
     pub coinbase_tx_value_remaining: u64,
     /// All the outputs that will be included in the coinbase txs
     #[cfg_attr(feature = "with_serde", serde(borrow))]
     pub coinbase_tx_outputs: B064K<'decoder>,
-    /// The locktime field in the coinbase transaction.
+    /// The `locktime` field in the coinbase transaction.
     pub coinbase_tx_locktime: u32,
     /// Merkle path hashes ordered from deepest.
     #[cfg_attr(feature = "with_serde", serde(borrow))]
     pub merkle_path: Seq0255<'decoder, U256<'decoder>>,
-    /// Size of extranonce in bytes that will be provided by the
-    /// downstream node.
+    /// Size of extranonce in bytes that will be provided by the downstream role.
     pub extranonce_size: u16,
 }
 
-/// # SetCustomMiningJob.Success (Server -> Client)
+/// Message used by upstream to accept [`SetCustomMiningJob`] request.
 ///
-/// Response from the server when it accepts the custom mining job. Client can start to mine on
-/// the job immediately (by using the job_id provided within this response).
+/// Upon receiving this message, downstream can start mining on the job immediately (by using the
+/// [`SetCustomMiningJobSuccess::job_id`] provided within this response).
```

Comment:
```suggestion
/// Upon receiving this message, downstream can start submitting shares for this job immediately (by using the
/// [`SetCustomMiningJobSuccess::job_id`] provided within this response).
```

correlated to https://github.com/stratum-mining/sv2-spec/issues/115

cc @GitGab19